### PR TITLE
various js and css cleanup

### PR DESCRIPTION
**Pull request by liuhelen10**
Minor JS and SCSS cleanup

### CODE REVIEW

**Comment by liuhelen10** [comment]
meow #fedex

File: src/webui/modules/home/index.js [hunk 1]
```diff
@@ -97,20 +97,20 @@ export default class Home extends React.Component {
   }
 
   isTherePackages() {
```
Comment (dschaller) [inline src/webui/modules/home/index.js]: super nit: can you change `isTherePackages` to `areTherePackages`?

**Review by dschaller — APPROVED** [review]

**Comment by liuhelen10** [comment]
🚀

File: static/index.html [hunk 2]
```diff
@@ -12,5 +12,5 @@
 <link rel="shortcut icon" href="ToReplaceByVerdaccio/-/static/favicon.ico"><link href="ToReplaceByVerdaccio/-/static/0.style.ff7923533d82cf5ff589.css" rel="stylesheet"><link href="ToReplaceByVerdaccio/-/static/2.style.dc4f731840d3f3151c12.css" rel="stylesheet"></head>
 <body class="body">
     <div id="root"></div>
-<script type="text/javascript" src="ToReplaceByVerdaccio/-/static/manifest.dcec93d13ceb4b12ce65.js"></script><script type="text/javascript" src="ToReplaceByVerdaccio/-/static/vendors.dcec93d13ceb4b12ce65.js"></script><script type="text/javascript" src="ToReplaceByVerdaccio/-/static/main.dcec93d13ceb4b12ce65.js"></script></body>
+<script type="text/javascript" src="ToReplaceByVerdaccio/-/static/manifest.21d29befa3a51c65c7a2.js"></script><script type="text/javascript" src="ToReplaceByVerdaccio/-/static/vendors.21d29befa3a51c65c7a2.js"></script><script type="text/javascript" src="ToReplaceByVerdaccio/-/static/main.21d29befa3a51c65c7a2.js"></script></body>
```
Comment (saponifi3d) [inline static/index.html:15]: we should make sure to add a ticket for this week to improve this workflow, so you don't have to build / commit the artifacts. :)

Comment (liuhelen10) [reply static/index.html:15]: yep! i have a ticket for it

**Pull request merged**